documentation/modules/auxiliary/scanner/http/wp_duplicator_file_read.md

```diff
@@ -0,0 +1,102 @@
+## Vulnerable Application
+The issue is being actively exploited, and allows attackers to download arbitrary files, such as the wp-config.php file.
```

Comment:
wrap wp-config.php in ticks (`) so it looks like `wipconfig.php`

Reply:
This line is also very similar to line 5, i'd pick one, and remove the other

Reply:
Yes, I fixed it